### DISCUSSION TITLE
Icon for BIMx

### DISCUSSION THREAD

**Discussion by oelamri** [elrumo/macOS_Big_Sur_icons_replacements#1290] [Icon request] [answered]
BIMx is the most popular CAD preview tool for architects. There is a MacOS app (https://graphisoft.com/solutions/bimx) and an iPad app (https://apps.apple.com/us/app/bimx/id452706864).

It's part of the popular ArchiCAD suite of products.

Can somebody please make a MacOS icon for BIMx that resembles its iPad app icon?

**Comment by ChilliChompa** [comment, marked as answer]
Hi, I have submitted one for approval. Cheers

![BIMx](https://user-images.githubusercontent.com/7188187/133389975-60c639dd-13f2-4670-a0d0-167adb995463.png)

**Reply by oelamri** [reply]
Nice! Looks great! Thanks @ChilliChompa!
